source/publish/push_repo.md

```diff
@@ -96,12 +96,94 @@ government services.
 ### GitHub Pages
 
 With some modification, you can also deploy your site with [GitHub
-Pages](https://pages.github.com/), but we do not support this. To do this, you
+Pages](https://pages.github.com/) [external link], but we do not support this. To do this, you
 could for example use the [`middleman-gh-pages`]
-tool](https://github.com/edgecase/middleman-gh-pages), which we do not
+tool](https://github.com/edgecase/middleman-gh-pages) [external link], which we do not
 support. We also cannot guarantee that all features of the tool will work if
 you deploy your site with GitHub Pages.
 
+### Example: Deploy your documentation to a temporary site on the GOV.UK PaaS
+
+You can deploy your documentation on a temporary site on the GOV.UK PaaS.
+
+This is useful for review of content by non-technical people, for example delivery managers or product owners.
+
+#### Before you start
+
+You must have a GOV.UK PaaS account. This account is called an organisation, or org. Sign up for an org account at [https://www.cloud.service.gov.uk/signup](https://www.cloud.service.gov.uk/signup).
```

Comment:
This is not always the case though. If the user is part of a team that already has an org, they can just get an account and be added to that org.

Reply:
Could this just say:

You must have a GOV.UK PaaS account. Sign up for an org account at [https://www.cloud.service.gov.uk/signup](https://www.cloud.service.gov.uk/signup).